setup.py

```diff
@@ -5,20 +5,22 @@
 setup(
     # Package
     name="python-dsconfig",
-    version="1.4.0",
-    packages=['dsconfig', 'dsconfig.appending_dict'],
+    version="1.5.0",
+    packages=["dsconfig", "dsconfig.appending_dict"],
     description="Library and utilities for Tango device configuration.",
     # Requirements
```

Comment:
Adding a `python_requires` line would be good.
For example: 
https://github.com/pypa/sampleproject/blob/150be5fc818484b78816529bc9ec59440a9bcdab/setup.py#L137

Reply:
Thanks for comments! I'll improve this PR.

Reply:
I included both mentioned PRs and added description